tests/cpp_validation_tests/test_validation.cpp

```diff
@@ -2,6 +2,13 @@
 //
 // SPDX-License-Identifier: MPL-2.0
 
+#include "doctest/doctest.h"
+#include "nlohmann/json.hpp"
```

Comment:
`nlohmann` is an external library and should be included as `<nlohmann/json.hpp>`